src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift

```diff
@@ -348,4 +357,36 @@ class QRScannerViewController: UIViewController {
 		previewLayer.mask?.addSublayer(throughHoleLayer)
 		previewLayer.mask?.addSublayer(backdropLayer)
 	}
+	
```

Comment:
And for warning on-behalf we do not need an action, correct?

Reply:
You need an event for warning on-behalf :)

Reply:
What I meant was that we do not need an UIActionSheet action here for scanning an event for the warning on-behalf?

Reply:
There is one in line 382 🕵️‍♂️

Reply:
🤦‍♂️